internal/stacks/stackruntime/internal/stackeval/stubs/unknown.go

```diff
@@ -223,9 +234,9 @@ func (u *unknownProvider) ReadDataSource(request providers.ReadDataSourceRequest
 }
 
 func (u *unknownProvider) CallFunction(request providers.CallFunctionRequest) providers.CallFunctionResponse {
-	// This is offline functionality, so we can hand it off to the unconfigured
-	// client.
-	return u.unconfiguredClient.CallFunction(request)
+	return providers.CallFunctionResponse{
+		Err: fmt.Errorf("CallFunction shouldn't be called on an unknown provider; this is a bug in Terraform - please report this error"),
```

Comment:
At least to me it's a bit confusing that we mix unknown and unconfigured in the error messages.

Reply:
Yeah, it should just say unknown for everything. Think it's fixed now.